test.js

```diff
@@ -55,8 +51,5 @@ module.exports = {
     "chai-friendly",
   ],
   globals: { ...mochaCakes2Globals },
-  env: {
-    ...env,
-    mocha: true,
-  },
+  env: { mocha: true },
```

Comment:
`node` and `es6` should also be set here

Reply:
If I haven't misunderstood the inheritance of these configs, these will be inherited from either `eslint-config-exp` or `eslint-config-exp/modules` since they are set in those.
Previously, all the `rules`, `parserOptions` and `env` was explicitly imported into this test config from `eslint-config-exp`. This made it impossible to use it together with the `eslint-config-exp/modules` config, so I thought it would be easier to rely on the parent-folder-inheritance to get around it.

An alternativt could be to add a `eslint-config-exp/es6-test` configuration.

Reply:
Since tests will run in a node environment and since we want to be able to use ES features in tests both the flags should probably be set. It seems weird to me that the environment setting is inherited from other extensions.

I'm afraid that this will break things since tests can be written using CommonJS while the source lib could be using ES modules.

Reply:
It is set.

The overrides extend from the base configs, so we are just adding `mocha` to the environment here.